install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -12,262 +12,728 @@ toc::[]
+useful) label.
+
+For these reasons, SCCs with *RunAsAny* for ID-related strategies should be
+protected so that ordinary developers do not have access to the SCC. On the
+other hand, SCC strategies set to *MustRunAs* or *MustRunAsRange* trigger ID
+validation (for ID related strategies), and cause default values to be supplied
+by OpenShift to the container when those values are not supplied directly in the
+pod definition or image.
+
+SCCs may define the range of allowed IDs (user or groups). If range checking is
+required (for example, using *MustRunAs*) and the allowable range is not defined
+in the SCC, then the project determines the ID range. Therefore, projects
+support ranges of allowable ID. However, unlike SCCs, projects do not define
+strategies, such as `*runAsUser*`.
+
+Allowable ranges are interesting not only because they define the boundaries for
+container IDs, but also because the minimum value in the range becomes the
+default value for the ID in question. For example, if the SCC ID strategy value
+is *MustRunAs*, and the minimum value of an ID range is 100, and the ID is
+absent from the pod definition, then 100 is provided as the default for this ID.
+
+As part of pod admission, the SCCs available to a pod are examined (roughly, in
+priority order followed by most restrictive) to best match the requests of the
+pod. Setting a SCC's strategy type to *RunAsAny* is less restrictive, whereas a
+type of *MustRunAs* is more restrictive. All of these strategies are evaluated.
+To see which SCC was assigned to a pod, use the `oc get pod` command:
 
-The pod security context (the `*SecurityContext*` stanza in a pod definition)
-provides three parameters which allow you to gain and control access to volumes:
-`*FSGroup*`,`*SupplementalGroups*`, and `*SELinuxOptions*`.
+====
+----
+# oc get pod <pod_name> -o yaml
+...
+metadata:
+  annotations:
+    openshift.io/scc: nfs-scc <1>
+  name: nfs-pod1 <2>
+  namespace: default <3>
+...
+----
+<1> Name of the SCC that the pod used (in this case, a custom SCC).
+<2> Name of the pod.
+<3> Name of the project. "Namespace" is interchangeable with "project" in OpenShift.
+See
+link:../architecture/core_concepts/projects_and_users.html#namespaces[Projects
+and Users] for details.
+====
+
+It may not be immediately obvious which SCC was matched by a pod, so the command
+above can be very useful in understanding the UID, supplemental groups, and
+SELinux relabeling in a live container.
+
+Any SCC with a strategy set to *RunAsAny* allows arbitrary values for that
+strategy to be defined in the pod definition (and/or image). When this applies
+to the user ID (`*runAsUser*`) it is prudent to restrict access to the SCC to
+prevent a container from being able to run as root.
+
+Because pods often match the *restricted* SCC, it is worth knowing the security
+this entails. The following observations apply to the *restricted* SCC:
+
+- Constrains user IDs due to the `*runAsUser*` strategy being set to
```

Comment:
I like "observations apply", personally, but it probably sounds better in an academic journal than here.  Suggest: "The *restricted* SCC has several characteristics", followed by complete sentences in the points.